sinks/kafka.go

```diff
@@ -117,12 +127,12 @@ func startConsumer(conf *kafkaSinkConfig, input chan *events.LookatchEvent, thre
 }
 
 // processGenericEvent process Generic Event
-func processGenericEvent(genericMsg *events.GenericEvent, conf *kafkaSinkConfig, threshold int) (*sarama.ProducerMessage, error) {
+func (k *Kafka) processGenericEvent(genericMsg *events.GenericEvent) (*sarama.ProducerMessage, error) {
 	var topic string
-	if len(conf.Topic) == 0 {
-		topic = conf.Topic_prefix + genericMsg.Environment
+	if len(k.kafkaConf.Topic) == 0 {
```

Comment:
I can't comment on the correct line because of github, but : it's kind of pointless to call your conf "kafkaConf" if it's in k, which is "* Kafka" type. "conf" alone would suffice.